code/Model/SiteTree.php

```diff
@@ -2246,11 +2246,13 @@ public function getCMSActions()
         }
 
         // "save", supports an alternate state that is still clickable, but notifies the user that the action is not needed.
+        $noChangesClasses = 'btn-outline-secondary font-icon-check-mark';
```

Comment:
does `btn-outline-secondary` need to be here if it's still hardcoded in the javascript?

Reply:
The javascript code is there for legacy reasons; This is the canonical source of these class names.